es_utils.py

```diff
@@ -267,3 +267,25 @@ def es_send_external_stats(stats_dict_file_path, opts_dict_file_path, cpu_normal
   sdata["@timestamp"]=file_stamp*1000
   try:send_payload(es_index_name+"-"+week, es_doc_name, index_sha, json.dumps(sdata))
   except Exception as e: print(e.message)
+
+def getExternalsESstats(arch='*', lastNdays=30, page_size=0):
+    stats = es_query(index='externals_stats_summary-*',
+                     query=format('architecture:%(architecture)s',
+                                  architecture=arch),
+                     start_time=1000*int(time()-(86400*lastNdays)),
+                     end_time=1000*int(time()),scroll=True)
```

Comment:
time() is not defined 
```
NameError: global name 'time' is not defined
```

Reply:
I added it, it wasn't available in es_utils.py (and I was using another script to run that function which has it, so didn't noticed)